Front/src/app/services/products.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ProductsModel } from '../models/ProductsModel';
+import { HttpResponseModel } from '../models/HttpResponseModel';
+import { AppConfig } from '../common/app-config';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductsService {
+
+  constructor(private htpp: HttpClient) { }
+
+  ServiceEndpoint: string = `${AppConfig.API_URL}`;
```

Comment:
You can concatenate the controller name here and avoid it in all the methods bellow. 
```suggestion
  ServiceEndpoint: string = `${AppConfig.API_URL}/api/Products`;
```